Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m

```diff
@@ -28,7 +28,15 @@
 #import "ADTestAppProfileViewController.h"
 #import "ADTestAppClaimsPickerController.h"
 
-@interface ADTestAppAcquireTokenViewController () <UITextFieldDelegate>
+#if AD_MAM_SDK_TESTING
+#import <IntuneMAMWalledGarden/IntuneMAM.h>
```

Comment:
Public release will be <IntuneMAM/IntuneMAM.h>

Reply:
Yep, will change that once I get the public MAM SDK with true mam changes from you :)